pkg/bundle/bundle_test.go

```diff
@@ -106,6 +106,39 @@ func Test_getBundleVersion(t *testing.T) {
 	}
 }
 
+func TestMinVersion(t *testing.T) {
+	t.Parallel()
+	for _, tc := range []struct {
+		name            string
+		mediaType       string
+		expectedVersion string
+		ret             bool
+	}{
+		{"old-format", "application/vnd.dev.sigstore.bundle+json;version=0.1", "v0.1", true},
+		{"old-format-unexpected", "application/vnd.dev.sigstore.bundle+json;version=0.1", "v0.2", false},
+		{"old-format-without-v", "application/vnd.dev.sigstore.bundle+json;version=0.1", "0.1", true},
```

Comment:
Do we want to support matching both with and without the preceding `v`, or should we standardize around one?

Reply:
I would favour the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle) here, what's implemented. 
As this is a simple helper function, requiring stricter formatting does not seem to aid in any correctness IMHO.